assets/js/components/common/BlankSlate.jsx

```diff
@@ -29,18 +29,22 @@ const styles = theme => ({
 
 @withStyles(styles)
 class BlankSlate extends Component {
+  constructor(props) {
```

Comment:
is this constructor needed for anything?

Reply:
oh right i dont need the constructor. It was broken before because we didnt' have this.props, just props